src/utils/getEtherscanLink.ts

```diff
@@ -5,31 +5,31 @@ export function getMultichainScanLink(
   data: string,
   type: "tx" | "token" | "address" | "block" | "txs",
 ): string {
-  let chainScanBaseName = "etherscan"
+  let chainScanDomain = "etherscan.io"
```

Comment:
Thank you for fixing this! Could I ask you one additional favor - Could you use `src/constants/networks.ts` to power this function? That way we have all data about networks and chainScans in one place.